### PR TITLE
Tushar

### PR DESCRIPTION
**Pull request by devcode1999**
Footer & About Page Work

### CODE REVIEW

File: views/about.ejs [hunk 1]
```diff
@@ -1 +1,121 @@
...
+        <h3>About the LADC Society</h3>
+        <br>
+        <p>The Signet Society is the place at Harvard where students and faculty who are interested in the arts can meet over lunch and 
+            without academic constraints. Special events such as readings, talks, screenings, shows, and receptions reflect the interests of 
+            undergraduates, graduate, and associate members alike: literature, the performing arts, the fine arts, the social sciences, politics, 
+            and culture. Renowned artists, writers, actors, and scholars visiting Cambridge often join Signet members for discussions or to give
+             more formal presentations. The Signet Society Alumni work to connect Signet alumni living in 35 countries with the Harvard community 
+             and each other.</p>
+    <br>
+    <br>
```
Comment (dhruvmig) [inline views/about.ejs]: Try using margin instead of multiple 'br' tags

Comment (devcode1999) [reply views/about.ejs]: ok sir, i will make the changes.

Comment (devcode1999) [reply views/about.ejs]: Done sir

File: views/about.ejs [hunk 2]
```diff
@@ -1,123 +1,144 @@
...
           </div>
-</div>
-<div class="col-10 col-md-3 col-sm-6 mb-2 col-lg-2 mb-5">
-    <div class="card">
-        <img src="https://scontent.fluh1-1.fna.fbcdn.net/v/t1.0-9/117385989_1676103219215429_6284840838078341627_o.jpg?_nc_cat=106&ccb=2&_nc_sid=8bfeb9&_nc_ohc=yOOdwACwpFcAX9Sb2IE&_nc_ht=scontent.fluh1-1.fna&oh=9b0e7ab4030ae5f3b617c33aae5a515e&oe=60300E66" class="card-img-top cimg" alt="...">
+        </div>
+      </div>
+    </div>
+    <div class="col-10 col-md-3 col-sm-6 mb-2 col-lg-2 marb-40">
+      <div class="card">
```
Comment (dhruvmig) [inline views/about.ejs]: height of the card is varying to the content size ,that makes length of each card different . try using text-wrap bootstrap class with some other variations to resolve this

Comment (devcode1999) [reply views/about.ejs]: Done Sir, I have made height responsive to the content and button will align itself to the bottom of the card.

**Comment by devcode1999** [comment]
I have updated my branch with the master and have committed to reflect the same

**Pull request merged**